website_text/for_authors.md

```diff
@@ -14,14 +14,26 @@ More details journal policies and procedures can be found in [our authors guide]
 
 # Before Submission
 
-Authors should first send a presubmission letter to an editor in the relevant area. 
+Authors **must** first send a presubmission letter to the section lead editor in the relevant area. 
 The contents of the  letter are described in the [Guide for writing articles for LiveCoMS](https://livecomsjournal.github.io/).
 **Submissions without an approved presubmission letter will be rejected without review, and authors will not receive any refund of their submission fee.**
 
 Once approved for submission, the authors should prepare the manuscript using [LaTeX](https://www.latex-project.org/) in a suitable public [GitHub](http://wwww.github.com) repository, using our [LaTeX template](https://github.com/livecomsjournal/article_templates). 
 Instructions on how to use the templates can be found [in the GitHub repository with the template files](http://https://github.com/livecomsjournal/article_templates). Submission for publication carries a $100 fee, the only fees associated with publication in LiveCoMS.
 
-A complete description of the submission process can be found in [the guide for writing an article for LiveCoMS](https://livecomsjournal.github.io/).
+A complete description of how to prepare an article can be found in [the guide for writing an article for LiveCoMS](https://livecomsjournal.github.io/).
+
+# Submission process
+
+Articles (after approval of the presubmission letter) are submitted on the journal site, from [the "For Authors" section](http://www.livecomsjournal.org/for-authors). When filling out the submission form, note the following instructions:
+
+* Even if a version of the article is posted on arXiv, do not select to import the submission information from arXiv.
+* Limit the abstract to 300 words, and should include the web address of the GitHub repository where the author's version of the article is hosted.
```

Comment:
Per other thread, I prefer not to set a length limit on the abstract unless there are strong reasons to; I'd ratehr have this taken care of by the review process (and note, the editor could comment too if he/she feels the abstract is too long).